tapdb/sqlc/queries/assets.sql

```diff
@@ -682,6 +682,7 @@ WITH target_asset(asset_id) AS (
             OR sqlc.narg('tweaked_script_key') IS NULL)
         AND (utxos.outpoint = sqlc.narg('outpoint')
             OR sqlc.narg('outpoint') IS NULL)
+    LIMIT 1
```

Comment:
Since the CTE seems to be the actual problem, should we change this query so it takes the asset primary DB ID? Since we're querying that ID in the next commit.

Reply:
We already have that query down below: `UpsertAssetProofByID`
Better to remove `UpsertAssetProof` and use two separate queries by default.